.github/workflows/manual-publish-docs.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  workflow_dispatch:
+
+name: Publish Documentation
+jobs:
+  build-publish:
+    runs-on: ubuntu-latest
+
+    services:
+      dynamodb:
+        image: amazon/dynamodb-local
```

Comment:
does this workflow actually need to build & test and use dynamoDB for publishing the docs? Not saying it's wrong - just wondering.